### PR TITLE
[PAN-OS] Add Ipv4/IPv6 sinkhole arguments

### PR DESCRIPTION
**Pull request by samuelFain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-7081](https://jira-hq.paloaltonetworks.local/browse/CIAC-7081)

## Description
PAN-OS's `pan-os-apply-dns-signature-policy` command returns error when `sinkhole` action is used, but not currently set for a given policy.
This PR adds the missing default IPv4/IPv6 sinkhole to http request parameters, and the option to set non-default IPv4/IPv6 sinkhole addresses with matching command arguments.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [x] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py
Comment (GuyAfik) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:7280]: @samuelFain don't you need this **only** in case the action is sinkhole? and not in **every** case?

File: Packs/PAN-OS/ReleaseNotes/1_17_12.md
Comment (ShacharKidor) [inline Packs/PAN-OS/ReleaseNotes/1_17_12.md:6]:
```suggestion
- Fixed an issue where the ***pan-os-apply-dns-signature-policy*** command failed when using the sinkhole action due to lack of default IPv4/Ipv6 sinkhole IP addresses.
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py
Comment (ShacharKidor) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:7280]: Do we have a TPB to make sure that all other commands pass successfully with the additional parameters' default values?

**Review by ShacharKidor — APPROVED** [review]
Nice! 
Let's wait for Guy's approval.

**Review by GuyAfik — APPROVED** [review]
@samuelFain nice!

File: Packs/PAN-OS/ReleaseNotes/1_17_12.md
Comment (samuelFain) [reply Packs/PAN-OS/ReleaseNotes/1_17_12.md:6]: Implemented suggestion.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py
Comment (samuelFain) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:7280]: There is a TPB that executing this command, but it is not among the TPBs that run during the build. I ran it separately and it passed successfully.
Also, it's important to note that these default values are sent with every command use-case, and they do not impact any other use-case of this command except when `action=sinkhole` is used. We have already tested this specific case to ensure its functionality.

**Pull request merged**
